superset-frontend/webpack.proxy-config.js

```diff
@@ -77,6 +78,13 @@ function toDevHTML(originalHtml) {
           return `<!-- DEV bundle: ${bundleName} ${assetType} START -->\n  ${(
             manifest.entrypoints[bundleName][assetType] || []
           )
+            .filter(chunkFilePath => {
+              if (loaded.has(chunkFilePath)) {
+                return false;
+              }
+              loaded.add(chunkFilePath);
+              return true;
+            })
```

Comment:
This is for fixing duplicate scripts in dev server proxy. Not related to this PR per se.